pkg/cmd/provisioning/gcp/create_service_accounts.go

```diff
@@ -124,7 +122,6 @@ func createServiceAccount(ctx context.Context, client gcp.Client, name string, c
 	if err != nil {
 		return "", errors.Wrap(err, "Error generating service account name")
 	}
-	serviceAccountName = serviceAccountName[:len(serviceAccountName)-6] // chop off the trailing random chars
```

Comment:
This was picked up from the actuator code. The `serviceAccountName` does not have any random characters to chop off IIUC. I would prefer longer names in the display without any truncated characters. Any thoughts?